tests/01-compile-base/Makefile

```diff
@@ -22,6 +22,7 @@ rpl-border-router/sky \
 slip-radio/sky \
 ipv6-hooks/sky \
 nullnet/native \
+mqtt-client/native \
```

Comment:
why `mqtt-client`? wasn't `dev/gpio-hal/` intended instead?

Reply:
This is as intended. Basically this pull enables 3 examples for native, but I only added this one to the tests. Do you see a point in adding the other two as well?

Reply:
Oh right, I see. Probably no point in adding the other two.